exercises/practice/word-count/test_word_count.c

```diff
@@ -17,6 +17,12 @@ void tearDown(void)
 {
 }
 
+static int cmp_word_count_word(const void *a, const void *b){
```

Comment:
*nit: this function could be named just `compare_word()`, the type of word is not super important to indicate because the function is private.

Reply:
Thanks, I've made the update.